packages/core/i18n/nls.zh-cn.json

```diff
@@ -1,25 +1,25 @@
 {
   "theia": {
     "callhierarchy": {
-      "open": "公开征集等级制度"
+      "open": "打开调用层次结构"
     },
     "core": {
       "common": {
-        "closeAll": "关闭所有标签",
-        "closeAllTabMain": "关闭主区域的所有标签",
-        "closeOtherTabMain": "关闭主区域的其他标签",
-        "closeOthers": "关闭其他标签",
-        "closeRight": "向右关闭标签",
-        "closeTab": "关闭标签",
-        "closeTabMain": "在主区域关闭标签",
-        "collapseAllTabs": "折叠 所有侧板",
-        "collapseBottomPanel": "拨动底部面板",
-        "collapseTab": "塌陷侧板",
-        "showNextTabGroup": "切换到下一个标签组",
-        "showNextTabInGroup": "切换到组中的下一个标签",
-        "showPreviousTabGroup": "切换到上一个标签组",
-        "showPreviousTabInGroup": "切换到组中的上一个标签",
-        "toggleMaximized": "切换到最大限度"
+        "closeAll": "全部关闭",
+        "closeAllTabMain": "关闭主区域的所有标签页",
+        "closeOtherTabMain": "关闭主区域的其他标签页",
+        "closeOthers": "关闭其他标签页",
+        "closeRight": "关闭右侧标签页",
+        "closeTab": "关闭标签页",
+        "closeTabMain": "关闭主区域标签页",
+        "collapseAllTabs": "折叠所有标签页",
+        "collapseBottomPanel": "折叠底部面板",
+        "collapseTab": "折叠标签页",
```

Comment:
Although the key is 'collapseTab', the English here is 'Collapse Side Panel'. Would it make sense to use a translation similar to 'Collapse Bottom Panel' above, something with '面板' rather than '标签页'?